problems/removeNumbersAtOddIndices.js

```diff
@@ -1,9 +1,13 @@
+const removeOddNumbers = require("./removeOddNumbers");
+
 /** Remove all numbers at odd indices from an array
 * @param {number[]} arr - The input array
 * @returns {number[]} - An array removing all elements initially appearing at an odd index
 */
-function removeNumbersAtOddIndices() {
-
+function removeNumbersAtOddIndices(arr) {
+    let ar = ['zero', 'one', 'two', 'three'];
```

Comment:
This ar isn't helping us. Don't create variables that won't help us. Think about if we were training for a customer service job. We care that we have customer, not that we have a specific customer. Our functions need to work for any input, in this case any array.